README.md

```diff
@@ -112,6 +121,16 @@ and then re-running the playbook.
 ```
 ansible-playbook -i /root/code/casl-ansible/inventory/sample.casl.example.com.d/inventory /root/code/casl-ansible/playbooks/openshift/end-to-end.yml
 ```
+### Start/Stop  a Cluster
+
+In the case we are using a Cloud Provider (AWS) to run the Cluster, it may be interesting to stop/start the instances where the Cluster is running in order to reduce costs. To do so, two playbooks are available, `start-cluster.yml` and `stop-cluster.yml`, re-using the same inventory:
+
+```
+ansible-playbook -i /root/code/casl-ansible/inventory/ample.aws.example.com.d/inventory /root/code/casl-ansible/playbooks/openshift/start-cluster.yml --private-key==<your_ssh_private_key_file>
```

Comment:
I think it's confusing to include the `--private-key` variable for these `AWS` related commands, but not the OpenStack ones. I'd suggest leaving it out, but maybe add a note somewhere else on the page about using the `--private-key` option if the default `id_rsa` isn't the key used

Reply:
Agree